Dolphin scripts/Entrance Randomizer/lib/entrance_rando.py

```diff
@@ -28,6 +27,17 @@ class Choice(IntEnum):
     LevelCRC.PENGUIN_TEMPLE,
 )
 
+one_way_exits = (
```

Comment:
Whilst we're moving stuff, knowing which exits are one-way isn't specific to the rando, you can move that out of `entrance_rando` and with `TEMPLES_WITH_FIGHT`, `SOFTLOCKABLE_ENTRANCES` if you want (don't have to do that in this PR, it was already here anyway)

I think it belongs even better directly encoded in `transition_infos`, but no need to scope creep on this PR.

Reply:
I don't want to do that right now.